aemedge/blocks/tabs/tabs.css

```diff
@@ -7,6 +7,11 @@
   justify-content: flex-start;
 }
 
+.tabs.wide .tabs-list {
```

Comment:
Above line 10 is where you can add your .tabs.wide specific style:
```
.tabs.wide {

h1,h2,h3,h4,h5,p {
text-align: center;
  }
}
```

Reply:
we can't do nesting of selectors in "standard css".